python/setup.py

```diff
@@ -12,5 +13,5 @@
     url='https://github.com/ROBOTIS-GIT/DynamixelSDK',
     author='Leon Jung',
     author_email='rwjung@robotis.com',
-    install_requires=['pyserial']
+    install_requires=['pyserial'] + (["msvcrt"] if platform.system() == 'Windows' else [])
```

Comment:
@roncapat 

msvcrt is for the product of microsoft?

Reply:
yeah

Reply:
@roncapat 

It says it couldn't find msvcrt package,
and following link says that it is pre-installed in windows
https://stackoverflow.com/questions/34580220/why-i-get-error-from-pip-installing-getch
how do you think about?